network-performance/network-performance-workflow.yaml

```diff
@@ -0,0 +1,164 @@
+          default: "\"\""
+          required: false
+        elasticsearch:
+          display:
+            description: The ElasticSearch service access parameters
+            name: ElasticSearch parameters
+          type:
+            type_id: list
+            items:
+              id: ElasticSearch
+              type_id: ref
+    ElasticSearch:
+      id: ElasticSearch
+      properties:
+        host: 
+          display:
+            description: The URL for the ElasticSearch host
+            name: ElasticSearch host URL
+          type:
+            type_id: string
+        username:
+          display:
+            description: The username for the ElasticSearch service
+            name: ElasticSearch username
+          type:
+            type_id: string
+        password:
+          display:
+            description: The password for the ElasticSearch service
+            name: ElasticSearch password
+          type:
+            type_id: string
+        index:
+          display:
+            description: The index for the ElasticSearch service
+            name: ElasticSearch index
+          type:
+            type_id: string
+steps:
+  uuidgen:
+    plugin: quay.io/arcalot/arcaflow-plugin-utilities:latest
+    step: uuid
+    input: {}
+  kubeconfig:
+    plugin: quay.io/arcalot/arcaflow-plugin-kubeconfig:latest
+    input:
+      kubeconfig: !expr $.input.kubeconfig
+  uperf_server:
+    plugin: quay.io/arcalot/arcaflow-plugin-uperf:latest
+    step: uperf_server
+    deploy:
+      type: kubernetes
+      connection: !expr $.steps.kubeconfig.outputs.success.connection
+      pod:
+        metadata:
+          namespace: default
+          labels:
+            arcaflow: uperf
+        spec:
+          pluginContainer:
+            imagePullPolicy: Always
+    input:
+      run_duration: 60
+      port: 20000
+  service:
+    plugin: quay.io/arcalot/arcaflow-plugin-service:latest
+    input:
+      connection: !expr $.steps.kubeconfig.outputs.success.connection
+      service:
+        metadata:
+          namespace: default
+        spec:
+          ports:
+           - name: control
+             port: 20000
+             protocol: TCP
+           - name: controludp
+             port: 20000
+             protocol: UDP
+           - name: workload
+             port: 30000
+             protocol: TCP
+           - name: workloadudp
+             port: 30000
+             protocol: UDP
+          selector:
+            arcaflow: uperf
+  uperf_client:
+    plugin: quay.io/arcalot/arcaflow-plugin-uperf:latest
+    step: uperf
+    deploy:
+      type: kubernetes
+      connection: !expr $.steps.kubeconfig.outputs.success.connection
+      pod:
+        spec:
+          pluginContainer:
+            imagePullPolicy: Always
+    input:
+      port: 20000
+      name: "netperf"
+      groups:
+        - nthreads: 1
+          transactions:
+            - iterations: 1
+              flowops:
+                - type: "accept"
```

Comment:
I don't know if it currently works with `accept`. `connect` was the one that worked in the demo.